packages/stores/src/query/fns/domain-status.ts

```diff
@@ -0,0 +1,67 @@
+import { ObservableCosmwasmContractChainQuery } from "../cosmwasm/contract-query";
+import { KVStore } from "@keplr-wallet/common";
+import { ChainGetter } from "../../common";
+import { computed } from "mobx";
+import { ObservableChainQueryMap } from "../chain-query";
+import { DomainStatus } from "./types";
+
+export class ObservableQueryDomainStatusInner extends ObservableCosmwasmContractChainQuery<DomainStatus> {
+  constructor(
+    kvStore: KVStore,
+    chainId: string,
+    chainGetter: ChainGetter,
+    protected override readonly contractAddress: string,
+    protected readonly domain: string
+  ) {
+    super(kvStore, chainId, chainGetter, contractAddress, {
+      get_domain_status: { domain: domain },
+    });
+  }
+
+  @computed
+  get domain_status(): any {
+    if (!this.response || !this.response.data.domain_status) {
+      return {};
+    }
+
+    return this.response.data.domain_status;
+  }
+  get registrationTime(): any {
+    if (
+      !this.response ||
+      !this.response.data.domain_status.Owned.registration_time
+    ) {
+      return {};
+    }
+
+    return this.response.data.domain_status.Owned.registration_time;
+  }
+}
```

Comment:
Avoid "any" typings. Also data, domain_status, owned can be undefined, so handle that as well.